keps/sig-storage/1790-recover-resize-failure/kep.yaml

```diff
@@ -0,0 +1,34 @@
+---
+title: Recover from volume expansion failure
+authors:
+  - "@gnuified"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-api-machinery
+reviewers:
+  - "@msau42"
+  - "@liggit"
+  - "@xing-yang"
+  - "@jsafrane"
+approvers:
+  - "@saad-ali"
+  - "@xing-yang"
+editor: TBD
+creation-date: 2020-01-27
+last-updated: 2020-01-27
```

Comment:
nit: This should be updated.

Reply:
fixed. But lost lgtm. can you add it back?